boot/util.ml

```diff
@@ -0,0 +1,77 @@
+(************************************************************************)
```

Comment:
No mli for this module?

Reply:
I marked it internal so I didn't care too much, but indeed we can add it; I expect most of this code to go eventually away.